packages/web-components/src/card/card.template.ts

```diff
@@ -0,0 +1,30 @@
+import { ElementViewTemplate, html, when } from '@microsoft/fast-element';
+import type { Card } from './card.js';
+
+/**
+ * The template for the Fluent Card web-component.
+ * @public
+ */
+export function cardTemplate<T extends Card>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template
+      role="group"
+      ?interactive="${x => x.interactive}"
+      ?selected="${x => x.selected}"
+      ?disabled="${x => x.disabled}"
```

Comment:
Do we need to set disabled on the template if it just maps to `aria-disabled`? It looks like the styles are mapping to `aria-disabled` (good), what would be the point of setting the disabled value here as well?